projects/plugins/crm/includes/ZeroBSCRM.DAL3.Obj.Contacts.php

```diff
@@ -3622,8 +3622,22 @@ private function tidy_contact($obj=false,$withCustomFields=false){
 
             // if have totals, pass them :)
             if (isset($obj->quotes_total)) $res['quotes_total'] = $obj->quotes_total;
-            if (isset($obj->invoices_total)) $res['invoices_total'] = $obj->invoices_total;
-            if (isset($obj->transactions_total)) $res['transactions_total'] = $obj->transactions_total;
+				if ( isset( $obj->invoices_total ) ) {
+					$contact                            = $zbs->DAL->contacts->getContact( $res['id'], array( 'withInvoices' => true ) ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					$deleted_invoice_details            = jetpackCRM_deleted_totals( $contact['invoices'] );
+					$res['invoices_total']              = $obj->invoices_total - $deleted_invoice_details['total'];
+					$res['invoices_total_with_deleted'] = $deleted_invoice_details['total'];
+					$res['invoices_count_with_deleted'] = $zbs->DAL->customer_has_count_obj_type( $res['id'], ZBS_TYPE_INVOICE, ZBS_TYPE_CONTACT ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
+					$res['invoices_count']              = $res['invoices_count_with_deleted'] - $deleted_invoice_details['count'];
+				}
+				if ( isset( $obj->transactions_total ) ) {
+					$contact                                = $zbs->DAL->contacts->getContact( $res['id'], array( 'withTransactions' => true ) ); // phpcs:ignore WordPress.NamingConventions.ValidVariableName.UsedPropertyNotSnakeCase
```

Comment:
I think this isn't needed due to earlier filtering of transactions?

Reply:
Taken out now, left a comment [with more info here](https://github.com/Automattic/jetpack/pull/30178#issuecomment-1531296759) :)